src/runtime_src/driver/xclng/drm/xocl/subdev/mm_qdma.c

```diff
@@ -253,18 +245,41 @@ static ssize_t qdma_migrate_bo(struct platform_device *pdev,
 		return -EIO;
 	}
 	sgt->nents = nents;
+	
+	ret = 0;
+	for_each_sg(sgt->sgl, sg, sgt->nents, i)
```

Comment:
Why do we need to go through sgl to compare the length here?  The sgl could be really long. I do not think we need to re-calculate the length.

Reply:
Yes, we can remove it. I put it there for cross-check.